winbuild/build.rst

```diff
@@ -42,8 +42,8 @@ behaviour of ``build_prepare.py``:
   If ``PYTHON`` is unset, the version of Python used to run
   ``build_prepare.py`` will be used. If only ``PYTHON`` is set,
   ``EXECUTABLE`` defaults to ``python.exe``.
-* ``ARCHITECTURE`` is used to select a ``x86`` or ``x64`` build. By default,
-  uses same architecture as the version of Python used to run ``build_prepare.py``.
+* ``ARCHITECTURE`` is used to select a ``x86``, ``x64`` or ``x86_arm64`` (for win-arm64) build.
```

Comment:
`x86_arm64` seems very odd to see... why not simply `arm64`?

Reply:
I agree it's strange.
It's coming from vcvarsall.bat (attached) and the fact the win-arm64 MSVC compiler is basically a cross-compiler, not native arm64. The host is working in x86 emulation mode.
[vcvarsall.bat.txt](https://github.com/python-pillow/Pillow/files/7483176/vcvarsall.bat.txt)

Reply:
Ah, got it. Thanks.

Reply:
Since this is an internal value only used by Pillow, there is nothing preventing the use of `arm64`. In fact, while `vcvarsall` uses `x86_arm64`, MSBuild uses `ARM64` (according to this PR, I haven't checked myself). The `x64` value already disagrees with `vcvarsall`'s `x86_amd64`. So I would also vote for either `arm64` or `ARM64`. Of the two, I would probably prefer `ARM64` to match `platform.machine()`, but either is fine.

Reply:
Good point, updated!